pax/plugins/io/WaveformSimulator.py

```diff
@@ -12,6 +12,7 @@
 
 from pax import plugin, units, utils
 
+import root_pandas
```

Comment:
You can add a `   # noqa` comment on the same line to make the code checker ignore it